server/src/controller/order.controller.ts

```diff
@@ -113,22 +168,44 @@ export const putRatedOrder = async (req: Request, res: Response) => {
     try {
 
         const { id, order_rated, master_id } = req.body
+
+        const readMasterRating = await Master.findOne({
+            attributes: ['rated_sum', 'rated_quantity'],
+            where: {
+                id: id
+            }
+        })
         
-        const readMasterRating = await db.query('SELECT rated_sum, rated_quantity FROM masters WHERE id = $1', [master_id])
-        
-        const { rated_sum, rated_quantity } = readMasterRating.rows[0]
-        
-        const newRatedSum = rated_sum + order_rated
-        
-        const newRatedQuantity = rated_quantity + 1
-        
-        const newRating = Number((newRatedSum / newRatedQuantity).toFixed(2)) 
-        
-        const updateMasterRating = await db.query('UPDATE masters SET rating = $1, rated_sum = $2, rated_quantity = $3 WHERE id = $4', [newRating, newRatedSum, newRatedQuantity, master_id])
-        
-        const updateRatedOrder = await db.query(`UPDATE orders SET order_rating = $1, uuid_id = 'Rated' WHERE id = $2`, [order_rated, id])
+        if( readMasterRating ) {
+            const { rated_sum, rated_quantity } = readMasterRating
         
-        res.status(200).json(updateRatedOrder.rows)
+            const newRatedSum = rated_sum + order_rated
+            
+            const newRatedQuantity = rated_quantity + 1
+            
+            const newRating = Number((newRatedSum / newRatedQuantity).toFixed(2)) 
```

Comment:
You should not round on the server side. Round the value on the client side please